pkg/commands/compute/build_test.go

```diff
@@ -16,20 +16,30 @@ import (
 	"github.com/fastly/cli/pkg/threadsafe"
 )
 
+// buildScriptRust is interpolated with each Rust test's inlined fastly.toml
+const buildScriptRust = "cargo build --bin {name} --release --target wasm32-wasi --color always"
+
+// buildScriptAssemblyScript is interpolated with each AssemblyScript test's
+// inlined fastly.toml
+const buildScriptAssemblyScript = "$(npm bin)/asc assembly/index.ts --outFile bin/main.wasm --optimize --noAssert"
+
+// buildScriptJavaScript is interpolated with each JavaScript test's inlined
+// fastly.toml
+const buildScriptJavaScript = "$(npm bin)/webpack && $(npm bin)/js-compute-runtime ./bin/index.js ./bin/main.wasm"
+
+// buildScriptGo is interpolated with each Go test's inlined fastly.toml
+const buildScriptGo = "tinygo build -target=wasi -wasm-abi=generic -gc=conservative -o bin/main.wasm ./"
```

Comment:
Since moving the build logic into the fastly.toml we needed to update a bunch of tests to have properly defined build steps.